Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -4,7 +4,7 @@
 #include <QFileDialog>
 #include <QToolButton>
 
-#include <Core/Utils/File/OBJFileManager.hpp>
+#include <Core/File/deprecated/OBJFileManager.hpp>

```

Comment:
The thing is it has to be adapted to the FileLoader interface, that's why it is in Core/File/deprecated instead of IO.